src/engine/sparqlExpressions/PrefilterExpressionIndex.h

```diff
@@ -130,12 +132,12 @@ using CompOp = valueIdComparators::Comparison;
 template <CompOp Comparison>
 class RelationalExpression : public PrefilterExpression {
  private:
-  // The ValueId on which we perform the relational comparison on.
-  ValueId referenceId_;
+  // The value that represents our reference value for filtering.
```

Comment:
You can be more explicit in the comment:
That value is the right hand side of the expression, so a `less than` expression with a value of `3` will represent the `?someVar < 3`.
You could additionally rename the variable to `rightHandSideValue_` or something. Then the code gets more readable.